lib/pages/AppDetails/AppDetails.tsx

```diff
@@ -323,12 +319,14 @@ const AppDetails: React.FC<AppDetailsProps> = ({
                 )}
               </div>
 
-              <ImageGallery
-                additionalClass={classes.appImageGallery}
-                items={imagesArray}
-                showNav={false}
-                showPlayButton={false}
-              />
+              {imagesArray.length !== 0 && (
```

Comment:
`!length` is enough
```suggestion
              {!imagesArray.length && (
```

Reply:
I understand the proposed solution, and it makes sense, but unfortunately, it generates an unwanted result.

With `imagesArray.length`, we are essentially checking for images. Unfortunately, `imagesArray.length` results in a `0` appearing on screen, and while the suggested solution of `!imagesArray.length` overrides the issue of having that `0` showing up, it causes a rectangle devoid of images to appear instead (this rectangle being the `ImageGallery` component rendered with no images).

The current solution of `imagesArray.length !== 0` causes none of the above to happen, so I suggest we keep it as is.

<img width="732" alt="Screenshot 2021-05-26 at 11 43 58" src="https://user-images.githubusercontent.com/67902809/119647453-31dd2080-be18-11eb-86bc-655aa44cc0c2.png">
<img width="736" alt="Screenshot 2021-05-26 at 11 46 41" src="https://user-images.githubusercontent.com/67902809/119647457-33a6e400-be18-11eb-946b-8533328e05c3.png">

Reply:
So you can simply cast it to a boolean or not?

Can you check this suggestion?
```suggestion
              {!!imagesArray.length && (
```

Anyway it works. No need to change. You'll see me using this suggestion more often.